priam/src/main/java/com/netflix/priam/defaultimpl/PriamConfiguration.java

```diff
@@ -1087,7 +1088,7 @@ public String getFlushInterval() {
 
     @Override
     public String getBackupStatusFileLoc() {
-        return config.get(CONFIG_BACKUP_STATUS_FILE_LOCATION,  "backup.status");
+        return config.get(CONFIG_BACKUP_STATUS_FILE_LOCATION,  getDataFileLocation() + File.separator + "backup.status");
```

Comment:
Why do you need `backup.status` here?

Reply:
This is an OSS implementation where we save backup/snapshot status in a file.

Reply:
Since we saw that default location of this file was also in current working directory, we decided to change it to data folder for sanity purposes.

Reply:
👍